src/components/FormTitle/FormTitle.jsx

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import "./FormTitle.scss";
+
+export const FormTitle = (props) => {
+    return (
+        <div className="formTitle">
+            <div className="formTitle__text">{props.text}</div>
```

Comment:
We never talked about this, but its more valuable to allow a title component to render `props.chldren` rather than a specific prop. This makes the whole component more flexible to what you put inside it. 

If you dont know what `props.children` are then look em up on the react website it will make sense. 

TLDR: Pass the title text via `props.children` and get rid of `props.text`.